lib/kubeclient/http_exception.rb

```diff
@@ -0,0 +1,20 @@
+# TODO: remove this on next major version bump
+class KubeException < StandardError
```

Comment:
@grosser don't you need the body here? (`initialize`, etc.)

Technically I suppose it works because you throw a `Kubeclient::HttpException` which contains the relevant attributes. Although to be strict, just by how this is defined, there could be `KubeException` with no `:error_code`, `:message`, `:response`, which is not backward compatible.